### ISSUE TITLE
spring.autoconfigure.exclude property parse error

### ISSUE THREAD

**Issue by jrbalsas** [AlexFalappa/nb-springboot#144]
application.properties editor complains when you try to exclude an Autoconfiguration class, however application compiles and can be deployed without problem, e.g.

`spring.autoconfigure.exclude=org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration
`

Error showed in netbeans editor: 
_Cannot parse 'org.springframework.boot.auconfigure.jdbc.DataSourceAutoConfiguration' as java.lang.Class_

![image](https://user-images.githubusercontent.com/356995/53567216-cd5e8880-3b5e-11e9-8eef-f3cce6094c2d.png)

**Comment by AlexFalappa** [comment]
@jrbalsas, yes some data types could require a special case treatment.  As you pointed out the standard logic is somehow faulty.

This is similar to #108 and #121.
